syncserver/tests.ini

```diff
@@ -13,7 +13,7 @@ use = egg:SyncServer
 public_url = http://localhost:5000/
 
 # This defines the database in which to store all server data.
-#sqluri = sqlite:////tmp/syncserver.db
+sqluri = sqlite:///:memory:
```

Comment:
I agree in general, but this file isn't used in production, and in fact this whole repo isn't used in production so I think it's OK in this instance.  If left unspecified it would default to an sqlite db in the current directory, which doesn't work in the docker build (I assume because it can't write to the current directory at runtime)